skiko/src/jvmMain/kotlin/org/jetbrains/skia/impl/Native.jvm.kt

```diff
@@ -78,6 +79,7 @@ actual open class InteropScope actual constructor() {
     actual inline fun <reified T> InteropPointer.fromInterop(decoder: ArrayInteropDecoder<T>): Array<T> =
         this@fromInterop as Array<T>
     actual fun toInteropForArraysOfPointers(interopPointers: Array<InteropPointer>): InteropPointer = interopPointers
+    actual fun toInterop(callback: (() -> Boolean)?) = callback?.let { BooleanSupplier(it) as Any }
```

Comment:
Not sure if this kind of conversion makes much sense, we could call `invoke()` on callback directly.

Reply:
I've added it to make sure it is an instance of `BooleanSupplier` and has `getAsBoolean` for JNI

Reply:
But why do we need BooleanSupplier here at all? Think it's an artefact of Java implementation. In Kotlin we have Function interface that does all we need.

Reply:
Replaced with `kotlin.jvm.functions.Function0.invoke()`